analyzers/src/Directory.Build.targets

```diff
@@ -7,6 +7,18 @@
     <BinariesFolder>$(MSBuildThisFileDirectory)\..\packaging\binaries\</BinariesFolder>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Using Include="System" />
+    <Using Include="System.Collections.Generic" />
+    <Using Include="System.Collections.Immutable" />
+    <Using Include="System.Diagnostics" />
```

Comment:
What was the way you decided which namespaces to include? For example `System.Diagnostics` has only 12 usages through our codebase, and I am not sure it is worth it adding here.

Reply:
For developer comfort reasons (speed of development). It's one of the namespaces that are in VB by default. It's a veeery practical to be able to type just `Debugger.Break()` or `Debugger.Launch()` anywhere, anytime.